Projects/UOContent/Mobiles/AI/BaseAI.cs

```diff
@@ -2730,7 +2738,7 @@ public virtual bool AcquireFocusMob(int iRange, FightMode acqType, bool bPlayerO
 
         eable.Free();
 
-        m_Mobile.FocusMob = newFocusMob;
+        m_Mobile.FocusMob = newFocusMob ?? enemySummonMob;
```

Comment:
Looks like this variable is never set at the same time as `newFocusMob`, so we probably can reuse `newFocusMob` instead of `enemySummonMob`?